pages/common/gh-auth.md

```diff
@@ -0,0 +1,24 @@
+# gh-auth
+
+> Authenticate with GitHub in GitHub cli.
+> More information: <https://cli.github.com/manual/gh_auth_login>.
+
+- Login with interactive prompt:
```

Comment:
Based on `aws-vault.md:
```suggestion
- Open a browser window and authenticate with github.com:
```
We can say `github.com` to make a difference with the enterprise server.

Reply:
When you type `gh auth login`, it opens prompt to ask if you want to paste the code or open browser (so not necessary "open browser"), and asks you if you want normal github or enterprise (so not necessary "with github.com")